### ISSUE TITLE
Error “” after login

### ISSUE THREAD

**Issue by albill** [urbit/urbit#5148]
I ignored my urbit for a month or so but saw it was up and running in a tab I kept open. Since the other day, whenever I authenticate to landscape, I see:

Error
“”

and it was "reconnecting..." forever in the upper left.

There is a stack trace of:

retrieve<T> (webpack:///src/logic/lib/useLocalStorageState.ts 4:25)
useLocalStorageState<T> (webpack:///src/logic/lib/useLocalStorageState.ts 16:43)
yh (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 159:52)
useLayoutEffect (webpack:///node_modules/react/cjs/react.production.min.js 25:280)
useLocalStorageState<T> (webpack:///src/logic/lib/useLocalStorageState.ts 16:28)
Content (webpack:///src/views/landscape/components/Content.tsx 41:4)
oh (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 153:145)
ji(a,b,c);I(M,M.current&1);b=$h (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 261:495)
Fj (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 246:267)
Fj (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 246:193)
yj (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 239:171)
Ig (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 230:136)
ck (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 281:42)
e (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 284:303)
Mj (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 240:119)
e (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 284:286)
flushSync (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js 290:292)
ReactDOM (webpack:///src/index.js 7:0)
__webpack_require__ (webpack:///webpack/bootstrap 19:21)
o (webpack:///webpack/bootstrap 83:9)
d (http://urbit.anonymous.computer/~landscape/js/bundle/index.c1a890b306daa1aee34b.js 1:915)

**Comment by lycurgus** [comment]
Does the same happen if you use another browser (or an Incognito/Private/etc window)?

What is the output of `+trouble` when run in the dojo (urbit's terminal)?

**Comment by albill** [comment]
Yes, it happens in multiple browsers (tried Firefox, Chrome, Safari).

Output is:
<pre>
[ [ %base-hash
    ~[0vl.mjv16.bdi95.nlarg.qlm6v.h8jus.s1fje.h0073.si89b.kdskq.gjnlf]
  ]
  [%sour-hash 0vl.mjv16.bdi95.nlarg.qlm6v.h8jus.s1fje.h0073.si89b.kdskq.gjnlf]
  [%home-hash 0vl.mjv16.bdi95.nlarg.qlm6v.h8jus.s1fje.h0073.si89b.kdskq.gjnlf]
  [%kids-hash 0vh.rh2u9.cvbvu.lu3c2.2hl2t.55shf.p4juj.j07er.ohpan.49qqh.ata7c]
  [ %glob-hash
    ~[
      [/~btc/js/bundle 0v7.v4dng.o33qi.kc497.5jc02.ke5es %done]
      [/~landscape/js/bundle 0v4.qs4qf.3bam7.736sg.0poq8.c6vhq %done]
    ]
  ]
  [%our ship=~poslet-montyl point='2156068645' life=[~ 1] rift=[~ 0]]
  [%sponsor ship=~fipdel point='65317' life=[~ 3] rift=[~ 1]]
  [%dopzod ship=~dopzod point='4608' life=[~ 3] rift=[~ 2]]
  "Compare lifes and rifts to values here:"
  "https://etherscan.io/address/azimuth.eth#readContract"
  "  life - getKeyRevisionNumber"
  "  rift - getContinuityNumber"
  ~
]
</pre>

**Comment by lycurgus** [comment]
Ok, you've got a weird base hash which seems to be coming from your star `~fipdel` (upstream, `~del`, looks to be okay). Not sure what's going on there but you've got the right `%glob-hash` for Landscape, so it's serving the right content which is good.

Is there anything reported in the dojo at the same time as the error in the browser?

**Comment by albill** [comment]
I wasn't seeing any output in dojo when the error comes up in the browser (which is when I load landscape) but... urbit just hung. When I couldn't get a response, I rebooted the entire server (computer) and tried to restart urbit again and now I get:

<pre>
~
urbit 1.5
boot: home is /root/urbit/postlet-montyl
loom: mapped 2048MB
lite: arvo formula 79925cca
lite: core 59f6958
lite: final state 59f6958
loom: mapped 2048MB
boot: protected loom
live: loaded: MB/452.263.936
home: bailing out
pier: serf unexpectedly shut down
</pre>

and a core dump file is in the directory afterwards.

**Comment by lycurgus** [comment]
Ok, that seems like a bigger problem - not sure what's going on there unfortunately. Was there an error prior to the hang? It sounds like the hard reboot might have corrupted something but I'm not sure how we can diagnose. Might need to wait for someone from Tlon to have a look.
